lib/services/suite/suite.go

```diff
@@ -1139,7 +1139,8 @@ func CollectOptions(opts ...Option) Options {
 	return suiteOpts
 }
 
-// ClusterConfig tests cluster configuration
+// ClusterConfig tests cluster configuration.
+// TODO in 8.0.0: Test only the individual resources.
```

Comment:
nit: I'd stick to `DELETE IN` phrasing to make it easy to search, and anyone should be able to delete it. Might want to reorganize it so that it's a simple `DELETE IN` without additional explanation.
```suggestion
// DELETE IN 8.0.0: Test only the individual resources.
```